app/connector/aws-ddb/src/test/java/io/syndesis/connector/aws/ddb/AWSDDBInsertItemTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.connector.aws.ddb;
+
+
+import org.junit.Ignore;
+
+@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+@Ignore("Make sure the AWSDDBConfiguration has the proper credentials before running this test")
```

Comment:
You should provide unit test with some embedded server. I had a quick look and it seems there are useful libraries for that: https://github.com/aws-samples/aws-dynamodb-examples/blob/master/src/test/java/com/amazonaws/services/dynamodbv2/DynamoDBLocalFixture.java - give it a try for next development iteration